    try {
%s%s        %s returnValue = %s(%s);
        %s;
        return SUCCESS;
    } catch (const std::exception &e) {
        OH_LOG_MESSAGE("%s Error: " + std::string(e.what()));
        result = 0;
        return FAIL;
    }
}

